extendr-api/src/scalar/rbool.rs

```diff
@@ -3,6 +3,9 @@ use crate::scalar::Scalar;
 use crate::*;
 use std::convert::TryFrom;
 
+/// Integer (`i32`) values that [`Rbool`] is allowed to take.
+const VALID_VALUES: [i32; 3] = [0, 1, i32::MIN];
```

Comment:
Is this `const` scoped to this file only? If so, that checks out. If not, I think it should be renamed something like `VALID_RBOOL_VALUES`

Reply:
I think so, otherwise it would have to be a `pub const`.